cli/src/main/java/com/devonfw/tools/ide/merge/xmlmerger/XmlMergeSupport.java

```diff
@@ -0,0 +1,332 @@
+   */
+  static void appendAttributes(StringBuilder sb, Element element, String attributePrefix, boolean includeMergeNs) {
+    NamedNodeMap attributes = element.getAttributes();
+    int attributeCount = attributes.getLength();
+    boolean separator = false;
+    for (int i = 0; i < attributeCount; i++) {
+      Attr attribute = (Attr) attributes.item(i);
+      if (includeMergeNs || !hasMergeNamespace(attribute)) {
+        if (separator) {
+          sb.append(" ");
+        } else {
+          separator = true;
+        }
+        sb.append(attributePrefix);
+        appendName(sb, attribute);
+        sb.append("='");
+        sb.append(escapeSingleQuotes(attribute.getValue()));
+        sb.append('\'');
+      }
+    }
+  }
+
+  /**
+   * @param value the {@link String} value.
+   * @return the given {@link String} with potential single quotes escaped for XML (e.g. to use in attributes).
+   */
+  static String escapeSingleQuotes(String value) {
+
+    return value.replace("'", "&apos;");
+  }
+
+  /**
+   * @param element the {@link Element}.
+   * @return the {@link QName} (URI + local name).
+   */
+  static QName getQualifiedName(Element element) {
+
+    String namespaceURI = element.getNamespaceURI();
+    String localName = element.getLocalName();
+    if (localName == null) {
+      localName = element.getTagName();
+    }
+    return new QName(namespaceURI, localName);
+  }
+
+  /**
+   * @param node the {@link Node} to check.
+   * @return {@code true} if the {@link Node} is a {@link Node#TEXT_NODE text node} or a {@link Node#CDATA_SECTION_NODE CDATA section}.
+   */
+  static boolean isTextual(Node node) {
+
+    short nodeType = node.getNodeType();
+    return (nodeType == Node.TEXT_NODE) || (nodeType == Node.CDATA_SECTION_NODE);
+  }
+
+  /**
+   * @param document the {@link XmlMergeSupport} from which to remove all {@link #hasMergeNamespace(Attr) merge namespace} {@link Attr attributes}.
+   */
+  static void removeMergeNsAttributes(XmlMergeDocument document) {
+
+    NodeList nodeList = document.getAllElements();
+    for (int i = nodeList.getLength() - 1; i >= 0; i--) {
+      Element element = (Element) nodeList.item(i);
+      removeMergeNsAttributes(element);
+    }
+  }
+
+  /**
+   * @param element the {@link Element} from which to remove {@link #hasMergeNamespace(Attr) merge namespace} {@link Attr attributes}.
+   */
+  static void removeMergeNsAttributes(Element element) {
+
+    removeAttriubtes(element, XmlMergeSupport::hasMergeNamespace);
+  }
+
+  /**
+   * @param element the {@link Element} from which to remove matching {@link Attr attributes}.
+   * @param filter the {@link Predicate} that {@link Predicate#test(Object) decides} if an {@link Attr attribute} shall be removed (if {@code true}) or
+   *     not.
+   */
+  static void removeAttriubtes(Element element, Predicate<Attr> filter) {
```

Comment:
Typo `removeAttriubtes`, please refactor.

Reply:
Adjusted.

Reply:
Dont know what went wrong here, but it was not fixed. I just fixed it even though this was already marked as resolved.